charts/cluster-autoscaler/Chart.yaml

```diff
@@ -17,4 +17,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.4.0
+version: 9.4.1
```

Comment:
Sorry to be a pain, but given we're bumping the default version of the CA, I'd rather make this bump a minor version rather than a patch, should increase the likelihood of those who aren't pinning it checking what's changed. 

(That's the last one, I promise, and thanks for your patience with this!)

Reply:
Yes agree, a minor version bump is more appropriate. Thanks for point it out.